contribs/emissions/src/main/java/org/matsim/contrib/emissions/ColdEmissionHandler.java

```diff
@@ -129,18 +124,19 @@ public void handleEvent(LinkLeaveEvent event) {
             }
         }
     }
-    static void handleNullVehicle( Id<Vehicle> vehicleId, EmissionsConfigGroup emissionsConfigGroup ){
+
+    static void handleNullVehicleECG(Id<Vehicle> vehicleId, EmissionsConfigGroup emissionsConfigGroup ){
         if( emissionsConfigGroup.getNonScenarioVehicles().equals( EmissionsConfigGroup.NonScenarioVehicles.abort ) ){
```

Comment:
Maybe this would bekome easier to read as a switch statement. But maybe it doesnt matter